Added Diffs to response to reviewers. (#20)
greenelab/word_lapse_manuscript: content/09.response-to-reviewers.md

>This paper develops a method to evaluate semantic shifts by calculating annual and interannual changes using multiple integrated models. This method achieves good results, but there are some shortcomings in the paper. There are some suggestions for revision.
>1.      The motivation is not clear. Please specify the importance of this paper.

**We have updated our introduction to make our motivation for this work more clear.

> Please see our inline changes below (Introduction Section below the Textual Changes header).**

Reviewer: ```suggestion
While we attempted to briefly paste the changes here, the introduction has changed so significantly that we recommend examining the new version.**
```

>2.      Please highlight the contributions of this paper.

**We have updated our introduction to make our contributions for this work more clear.

> Please see our inline changes below (Introduction Section below the Textual Changes header).**

Reviewer: ```suggestion
As above, the introduction has changed so substantially that it was less efficient to convey this by pasting specific paragraphs here than it is to refer to the new introduction itself.**
```

>3.      Most of references are out of date. Please discuss more recently published solutions, especially the solutions published in 2022.

**We have updated our introduction to include more recent publications that pertain to this work.

> Please see our inline changes below (Introduction Section below the Textual Changes header).**

Reviewer: ```suggestion
**
```

>•       The introduction is not clear and very less literature is used. Follow this instruction: The introduction should briefly place the study in a broad context and highlight why it is important. It should define the purpose of the work and its significance, including specific hypotheses being tested. The current state of the research field should be reviewed carefully, and key publications cited. Please highlight controversial and diverging hypotheses when necessary. Finally, briefly mention the main aim of the work and highlight the main conclusions. Keep the introduction comprehensible to scientists working outside the topic of the paper.
>•       In the introduction, what key theoretical perspectives and empirical findings in the main literature have already informed the problem formulation? What major, unaddressed puzzle, controversy, or paradox does this research address?

**We have updated our introduction to include more recent publications and discussion in relation to this work.

> Please see our inline changes below (Introduction Section below the Textual Changes header).**

Reviewer: ```suggestion
**
```

>•       Authors should further clarify and elaborate novelty in their contribution.

**We have updated our introduction to make our contributions for this work more clear.

> Please see our inline changes below (Introduction Section below the Textual Changes header).**

Reviewer: ```suggestion
We attempted to provide a concise example of changes, but the changes were so extensive that it is more effective to simply point the reviewer to the new introduction.**
```

>4. The introduction has few quotes, it is very short.

**We have extended our introduction to include a more thorough discussion on previous work and highlight our contributions/motivation.

> Please see our inline changes below (Introduction Section below the Textual Changes header).**

Reviewer: ```suggestion
We now highlight related work both within and outside biomedicine.**
```

>5.      Provide a separate discussion section that explains the complete details of the evaluation of the word meanings.

**We updated our discussion per the given feedback.

> Please see our inline changes below (Discussion Section below the Textual Changes header).**

Reviewer: ```suggestion
The changes are extensive enough that we refer the reviewer to our new Discussion section.**
```

>8.      The authors should remove the citations from the Conclusion section.

**We have removed the citation from our conclusion section.

> Please see our inline changes below (Conclusion Section below the Textual Changes header).**

Reviewer: ```suggestion
We also condensed the section with updates to the Discussion section. The conclusions section now reads:
> We uncovered semantic changes within biomedical literature using a novel approach that accounts for inter- and intra-year variability.
> Our approach found 41,000 changepoints that include well-known examples such as 'cas9', 'pandemic', and 'sars'.
> We constructed a web application that allows users to manually examine these individual changepoints.
> As an extension to this project, future work may be able to determine the consistency and time-lag of semantic change between preprint and pre-publication peer-reviewed text - potentially predicting future change in pre-publication peer-reviewed text.
> Furthermore, including other preprint repositories may reveal consistencies across a broader swath of fields, or within-field analyses may reveal the earliest starting points of semantic changes that ultimately sweep through biomedicine.
> Overall, this work is one starting point regarding semantic change within biomedical literature, and we are excited to see how this landscape will change as time progresses.
**
```
